e2e-test-app/src/main/java/com/sap/ai/sdk/app/controllers/ScenarioController.java

```diff
@@ -34,6 +34,6 @@ AiScenarioList getScenarios() {
   @GetMapping("/models")
   @Nullable
   public AiModelList getModels() {
-    return API.modelsGet("foundation-models", "default");
+    return API.get_0("foundation-models", "default");
   }
```

Comment:
(Major)

This is a problem.

Cause: Following operations resolve to the same method names + signature `(String, String)`:

```yaml
get:
  tags:
  - Scenario
  summary: Get information about all models available in LLM global scenario
  description: Retrieve information about all models available in LLM global scenario
  operationId: models.get
```

```yaml
get:
  tags:
  - Scenario
  summary: Get scenario by id
  description: Retrieve details for a scenario specified by scenarioId.
  operationId: scenario.get
```

[If it was fixed](https://github.wdf.sap.corp/AI/apis/blob/master/ai_api.yaml#L906), we could use the logic just fine:

 ```diff
- operationId: models.get
+ operationId: scenario.get_models
```

I created [a PR to address this inconsistency here](https://github.wdf.sap.corp/AI/ml-api-facade/pull/1242)

Reply:
The fix has been merged